### DISCUSSION TITLE
🚀 Feature request fixed_order param for relationship tags

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#3317] [Feature Requests]
It sounds a bit odd, as you can order relationships in the entry, but I've got a case where user wants to use low reorder entry list to set the order on the relationship display on the frontend, and the fixed_order param isn't available (and doesn't work if you just stick it in there).  I think it's not something highly in demand, but it's handy to have and I'm hoping not difficult to add.

**Comment by creativearc-paul** [comment]
Giving this a thumbs up, also so I get update notifications =)

**Comment by intoeetive** [comment]
https://github.com/ExpressionEngine/ExpressionEngine/pull/4100
